platform/base/core/pom.xml

```diff
@@ -58,7 +58,9 @@
                             com.peregrine.replication,
                             com.peregrine.transform,
                             com.peregrine.transform.*,
-                            com.peregrine.commons.util
+                            com.peregrine.commons.util,
+                            com.peregrine.model.api,
+                            com.peregrine.model.impl
```

Comment:
do we really need to export the impl here?

Reply:
#671 handles that